src/main/groovy/edu/oregonstate/mist/cardsapi/resources/CardsResource.groovy

```diff
@@ -34,10 +28,15 @@ import org.skife.jdbi.v2.DBI
 @PermitAll
 @Produces(MediaType.APPLICATION_JSON)
 class CardsResource extends Resource {
-    Logger logger = LoggerFactory.getLogger(CardsResource.class)
 
     private final CardDAO cardDAO
     private DBI dbi
+    CardFluent cardFluent = new CardFluent(dbi)
```

Comment:
Similar to the CardDao, I don't think we should create the CardFluent object here. Let's create it in the base application class then you can pass the cardFluent object to the constructor of the resource class.